src/Orleans/Streams/PersistentStreams/StreamQueueBalancerType.cs

```diff
@@ -1,19 +1,22 @@
 namespace Orleans.Streams
 {
-    public enum StreamQueueBalancerType
+    /// <summary>
+    /// Built-in stream queue balancer type which is supported natively in orleans
+    /// </summary>
+    public static class BuiltInStreamQueueBalancerType
```

Comment:
?? if you are worrying about breaking change, only by changing the name would make this a breaking change. Change it from a  enum class to a static class with strings won't make it a breaking change. By breaking change I mean: user will have compiling errors from it. 

And I don't fully understand the logic behind an extra QueueBalancer setting. To avoid breaking change? Then back to my original question. 

I guess my question is, what is your concern? if you are worrying about breaking change, then I should just change the name back, right?